app/code/VConnect/Blog/Controller/Adminhtml/Post/Edit.php

```diff
@@ -0,0 +1,79 @@
+<?php
+declare(strict_types=1);
+
+namespace VConnect\Blog\Controller\Adminhtml\Post;
+
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use VConnect\Blog\Controller\Adminhtml\Post;
+use Magento\Framework\View\Result\PageFactory;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\Controller\ResultInterface;
+use VConnect\Blog\Api\Data\PostInterfaceFactory;
+
+/**
+ * Edit Blog Post action.
+ */
+class Edit extends Post implements HttpGetActionInterface
+{
+    /**
+     * @var \Magento\Framework\View\Result\PageFactory
+     */
+    protected $resultPageFactory;
+
+    /**
+     * Factory to create Post Model Object
+     * * @var \VConnect\Blog\Api\Data\PostInterfaceFactory
+     */
+    private PostInterfaceFactory $postInterfaceFactory;
+
+
+    /**
+     * Edit constructor.
+     * @param Context $context
+     * @param PageFactory $resultPageFactory
+     * @param PostInterfaceFactory $postInterfaceFactory
+     */
+    public function __construct(
+        Context $context,
+        PageFactory $resultPageFactory,
+        PostInterfaceFactory $postInterfaceFactory
+    ) {
+        $this->resultPageFactory = $resultPageFactory;
```

Comment:
i think you can inject all your class properties from constructor arguments using `private PostInterfaceFactory $postInterfaceFactory` and remove property declaration inside constructor and above for ALL classes